webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js

```diff
@@ -5,6 +5,11 @@ var SignatureCorrelations = (function() {
     var container = $('#mainbody');
     return {
         showSignatureCorrelationsTab: function() {
+            if (!container.data('product')) {
```

Comment:
You're verifying that ``product`` is false, but in the comment you talk about ``platform``. Which one holds the truth?